### PR TITLE
Update for RDNN

### LINKED ISSUE #72: Some panes does not displayed
screenshot from 2018-06-09 18-35-58

Version: Juno (daily)

Miscellaneous, Launcher, and Cerbere pane does not displayed.
I think this is due to the migration to RDNN which is progressing on elementary OS.

### PR DESCRIPTION
**Pull request by mdh34**
Fixes #72

### CODE REVIEW

File: src/Panes/CerberePane.vala [hunk 1]
```diff
@@ -25,7 +25,7 @@ namespace ElementaryTweaks {
         }
 
         construct {
-            if (Util.schema_exists ("org.pantheon.desktop.cerbere")) {
+            if (Util.schema_exists ("io.elementary.desktop.cerbere")) {
```
Comment (ryonakano) [inline src/Panes/CerberePane.vala]: I suppose we can change to `if (Util.schema_exists ("org.pantheon.desktop.cerbere") || Util.schema_exists ("io.elementary.desktop.cerbere")) {`

File: src/Panes/MiscPane.vala [hunk 2]
```diff
@@ -26,7 +26,7 @@ namespace ElementaryTweaks {
         }
 
         construct {
-            if (Util.schema_exists ("org.pantheon.desktop.wingpanel.indicators.sound")) {
+            if (Util.schema_exists ("io.elementary.desktop.wingpanel.sound")) {
```
Comment (ryonakano) [inline src/Panes/MiscPane.vala]: `if (Util.schema_exists ("org.pantheon.desktop.wingpanel.indicators.sound") || Util.schema_exists ("io.elementary.desktop.wingpanel.sound")) {`

File: src/Panes/SlingshotPane.vala [hunk 3]
```diff
@@ -31,7 +31,7 @@ namespace ElementaryTweaks {
         }
 
         construct {
-            if (Util.schema_exists ("org.pantheon.desktop.slingshot")) {
+            if (Util.schema_exists ("io.elementary.desktop.wingpanel.applications-menu")) {
```
Comment (ryonakano) [inline src/Panes/SlingshotPane.vala]: `if (Util.schema_exists ("org.pantheon.desktop.slingshot") || Util.schema_exists ("io.elementary.desktop.wingpanel.applications-menu")) {`

File: src/Settings/CerbereSettings.vala [hunk 4]
```diff
@@ -27,7 +27,7 @@ namespace ElementaryTweaks {
         static CerbereSettings? instance = null;
 
         private CerbereSettings () {
-            base ("org.pantheon.desktop.cerbere");
+            base ("io.elementary.desktop.cerbere");
```
Comment (ryonakano) [inline src/Settings/CerbereSettings.vala]: `base ((Util.schema_exists ("io.elementary.desktop.cerbere"))?"io.elementary.desktop.cerbere":"org.pantheon.desktop.cerbere");`

File: src/Settings/IndicatorSoundSettings.vala [hunk 5]
```diff
@@ -39,7 +39,7 @@ namespace ElementaryTweaks {
 
         public static Settings get_settings () {
             if (settings == null) {
-                settings = new Settings ("org.pantheon.desktop.wingpanel.indicators.sound");
+                settings = new Settings ("io.elementary.desktop.wingpanel.sound");
```
Comment (ryonakano) [inline src/Settings/IndicatorSoundSettings.vala]: `settings = new Settings ((Util.schema_exists ("io.elementary.desktop.wingpanel.sound"))?"io.elementary.desktop.wingpanel.sound":"org.pantheon.desktop.wingpanel.indicators.sound");`

File: src/Settings/SlingshotSettings.vala [hunk 6]
```diff
@@ -28,7 +28,7 @@ namespace ElementaryTweaks {
 
         private SlingshotSettings ()
         {
-            base ("org.pantheon.desktop.slingshot");
+            base ("io.elementary.desktop.wingpanel.applications-menu");
```
Comment (ryonakano) [inline src/Settings/SlingshotSettings.vala]: `base ((Util.schema_exists ("io.elementary.desktop.wingpanel.applications-menu"))?"io.elementary.desktop.wingpanel.applications-menu":"org.pantheon.desktop.slingshot");`

**Comment by Philip-Scott** [comment]
@ryonakano Not really due to Loki not having some settings Juno has, and so on. So I'll just stop Loki builds for now :) 

@mdh34 Hey! I'm closing the PR.... I completely forgot about it and just pushed the RDNN changes to master (As well as removing the Cerbere pane since that's not a thing anymore) :grimacing: Sorry about that :/

**Pull request closed without merging**